src/vs/workbench/contrib/preferences/browser/media/settingsWidgets.css

```diff
@@ -27,6 +27,17 @@
 	margin-left: 4px;
 	min-width: 40%;
 }
+.settings-editor > .settings-body > .settings-tree-container .setting-item.setting-item-list .setting-list-object-widget .setting-list-object-input-key-checkbox {
+	margin-left: 4px;
+	height: 24px;
+}
+.settings-editor > .settings-body > .settings-tree-container .setting-item-bool .setting-list-object-input-key-checkbox .setting-value-checkbox {
+	margin-top: 3px;
+}
+.settings-editor > .settings-body > .settings-tree-container .setting-item-bool .setting-list-object-value .value-description {
+	font-style: italic;
```

Comment:
I'm not crazy about the italic styling, to me it doesn't match anything else in vscode